### DISCUSSION TITLE
How to reset the infinite grab range back to normal?

### DISCUSSION THREAD

**Discussion by feuer007** [winstxnhdw/lc-hax#414] [Q&A]
I'm using this hack mainly for its anti-kick function and try to be as close to vanilla as possible, so I deleted the corresponding scripts and disabled the loading of corresponding mods.
However I couldn't figure out how to how to reset the grab range back to normal.
Something I found was a "Range" file in (...)/lc-hax/lc-hax/Scripts/Extensions, but I'm not sure how to correctly modify it.

Any help will be appreciated :)

**Comment by winstxnhdw** [comment]
Stick to only deleting what is in Mods and the Patches folder and nowhere else.

**Reply by winstxnhdw** [reply]
Keep the Patches/Dependencies folder too.
